.mergify.yml

```diff
@@ -1,15 +1,43 @@
 extends: .github
+shared:
+  check_runs: &CheckRuns
+    - check-success=test
+
+merge_protections_settings:
+  reporting_method: deployments
+
+merge_protections:
+  - name: 🤖 Continuous Integration
+    if: []
+    success_conditions:
+      - and: *CheckRuns
+
+  - name: 👀 Review Requirements
+    if: []
+    success_conditions:
+      - or:
+        - "#approved-reviews-by>=2"
+        - author = dependabot[bot]
+        - author = renovate[bot]
+        - and:
+            - author = mergify-ci-bot
+            - head = openapi-spec-sync
+        - and:
+            - author = mergify-ci-bot
+            - head ~= ^clifus/
+            - "title ~= ^chore: bump"
 
 queue_rules:
   - name: default
     allow_inplace_checks: true
     queue_conditions:
-      - and: &CheckRuns
-          - check-success=test
-      - "#approved-reviews-by>=2"
-      - "#changes-requested-reviews-by=0"
-      - "#review-threads-unresolved=0"
-      - "#review-requested=0"
+      # We have to set this until we can put the queue below before so it
```

Comment:
Fine though wondering for this repo if we need 2 queues, we can keep things simple.